src/sparseml/pytorch/image_classification/utils/trainer.py

```diff
@@ -233,8 +233,14 @@ def _apply_one_shot(self):
         _LOGGER.info(f"Applied {self.recipe_path} to manager")
 
     def _initialize_module_tester(self):
+        """
+        This uses self.model.module unlike the ModuleTrainer which uses self.model
+        so that the validation runs on the local copy of the model on the rank 0 gpu
+        rather than on the distributed model
```

Comment:
nit: for private methods, use in-line comments (`# `) instead of block. we generally standardize to reserving block comments for top level file stuff and pydocs